install.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+if [[ $TRAVIS_OS_NAME == 'osx' ]]; then
+    case "${TOXENV}" in
+        py35)
+            # Install Python 3.5 from source
+            wget https://www.python.org/ftp/python/3.5.2/Python-3.5.2.tar.xz
+            tar -xf Python-3.5.2.tar.xz
+            cd Python-3.5.2
+            ./configure
+            make
+            make install
+            cd ..
+            rm -rf Python-3.5.2
+        ;;
+        py34)
+            # Install Python 3.4 from source
+            wget https://www.python.org/ftp/python/3.4.3/Python-3.4.3.tar.xz
+            tar -xf Python-3.4.3.tar.xz
+            cd Python-3.4.3
+            ./configure
+            make
+            make install
+            cd ..
+            rm -rf Python-3.4.3
```

Comment:
Not a shell scripter ... which is why I love xonsh!

Any help here would be appreciated.

Installing from source b/c I don't know how to make brew can install arbitrary python3 options ... and I'd rather not also install something like pyenv ...